src/v2/Components/Cells/PartnerCell.tsx

```diff
@@ -41,6 +41,9 @@ const PartnerCell: React.FC<PartnerCellProps> = ({
   const locations = extractNodes(partner.locationsConnection)
   const meta = uniq(locations.map(location => location.city?.trim())).join(", ")
   const image = partner.profile?.image?.cropped
+  const badgedPartners = partner.categories
+    ?.filter(category => category?.slug === "black-owned")
+    .map(category => ({ children: category?.name }))
```

Comment:
A nice follow up would be to add a `displayable` (or something) argument to the Metaphysics field and do the filtering there.

Reply:
Created a JIRA ticket to discuss the following proposed MP transition.